dynawo/sources/Solvers/Common/DYNSolverImpl.cpp

```diff
@@ -165,7 +165,7 @@ Solver::Impl::printHeader() const {
 
 void
 Solver::Impl::printSolve() const {
-  std::stringstream msg;
+  stringstream msg;
```

Comment:
missing 
msg.str(""); 
msg.clear();

Reply:
not required for this one as it is not in a loop

Reply:
ok!